.github/e2e-test/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM node:16-slim
```

Comment:
Any reason to use a complex image instead of a simple one (e.g. `alpine`)?

Reply:
Switched to alpine